app/assets/stylesheets/pages/components/_variables.scss

```diff
@@ -0,0 +1,11 @@
+
+
+@mixin bg_section{
```

Comment:
If you don't need params it would be a better approach to use the [Extend/Inheritance](https://sass-lang.com/guide)
feature!